.prettierignore

```diff
@@ -0,0 +1,13 @@
+*
+!*/
+!*.js
+!*.jsx
+!*.json
+!*.scss
+_site
+package-lock.json
+package.json
```

Comment:
```suggestion
playground.js.map
playground.js
debug*.json
```
We can use prettier to format the `package.json` file. Any reason for ignoring it?